perf/load/auto-mtls/templates/config-changer.yaml

```diff
@@ -77,17 +77,13 @@ spec:
       containers:
       - args:
         - /script/scale.py
-        - --sidecar-name
-        - {{ .Values.serviceNamePrefix }}0-automtls-sidecar
-        - --nosidecar-name
-        - {{ .Values.serviceNamePrefix }}0-automtls-nosidecar
         env:
         # Needed to flush the output for python in detached container mode.
         - name: PYTHONUNBUFFERED
           value: "0"
-        image: gcr.io/jianfeih-test/hyperkube:v1.12.1
-        imagePullPolicy: IfNotPresent
-        name: config-change
+        image: gcr.io/jianfeih-test/automtls-test:1113a
```

Comment:
is it pushed to istio hub?

Reply:
which hub, `gcr.io/istio-testing`?

Reply:
Done. I have the permission, not sure if it's intended though